lib/Runtime/Types/EdgeJavascriptTypeId.h

```diff
@@ -120,6 +120,7 @@ enum TypeId
     TypeIds_ActivationObject = 73,
     TypeIds_SpreadArgument = 74,
     TypeIds_ModuleNamespace = 75,
+    TypeIds_ListIterator = 76,
```

Comment:
does it matter?? I am a bit hesitate to change order unless necessary...

---
In reply to: [74170867](https://github.com/Microsoft/ChakraCore/pull/1401#discussion_r74170867) [](ancestors = 74170867)